phys2denoise/cli/run.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+"""Parser for phys2denoise."""
+
+
+import argparse
+
+from phys2denoise import __version__
+
+
+def _get_parser():
+    """
+    Parse command line inputs for this function.
+
+    Returns
+    -------
+    parser.parse_args() : argparse dict
+
+    Notes
+    -----
+    # Argument parser follow template provided by RalphyZ.
+    # https://stackoverflow.com/a/43456577
+    """
+    parser = argparse.ArgumentParser()
+    optional = parser._action_groups.pop()
+    metric = parser._action_groups.pop()
+    required = parser.add_argument_group('Required Argument:')
+    required.add_argument('-in', '--input-file',
+                          dest='filename',
+                          type=str,
+                          help='Path/name of the file containing physiological '
+                               'data, with or without extension.',
+                          required=True)
+    metric.add_argument('-crf', '--crf',
+                        dest='metric_list',
+                        action='append_const',
+                        const='crf',
+                        help='Cardiac response function. Needs the following '
+                             'inputs:sr, os, tl, onset and tr.',
+                        default=False)
+    metric.add_argument('-rpv', '--rpv',
+                        dest='metric_list',
+                        action='append_const',
+                        const='rpv',
+                        help='REspiratory pattern variability. Needs the following '
+                             'inputs: bts and win.',
+                        default=False)
+    metric.add_argument('-env', '--env',
+                        dest='metric_list',
+                        action='append_const',
+                        const='env',
+                        help='Respiratory pattern variability calculated across a sliding '
+                             'window. Needs the following inputs: bts, sr, osr, win and lags.',
+                        default=False)
+    metric.add_argument('-rv', '--rv',
+                        dest='metric_list',
+                        action='append_const',
+                        const='rv',
+                        help='Respiratory variance. Needs the following inputs: '
+                             'bts, sr, osr, win and lags.',
+                        default=False)
+    metric.add_argument('-rvt', '--rvt',
+                        dest='metric_list',
+                        action='append_const',
+                        const='rvt',
+                        help='Respiratory volume-per-time. Needs the following inputs: '
+                             'bts, sr, osr, win and lags.',
+                        default=False)
+    metric.add_argument('-rrf', '--rrf',
```

Comment:
I don't think there's anything to be gained from having long and short versions of the same argument if they're the same string. In general, I'd drop all of the short versions except for cases where there's a very good reason to have one.

Reply:
I prefer short versions to long versions tbh (hey, one dash more? too much typing already).
Jokes aside, we can use full names of metrics in the long versions.